ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/AffectedDeploymentsTable.tsx

```diff
@@ -94,87 +97,94 @@ function AffectedDeploymentsTable({
                     <Th>First discovered</Th>
                 </Tr>
             </Thead>
-            {deployments.length === 0 && <EmptyTableResults colSpan={7} />}
-            {deployments.map((deployment, rowIndex) => {
-                const {
-                    id,
-                    name,
-                    namespace,
-                    clusterName,
-                    lowImageCount,
-                    moderateImageCount,
-                    importantImageCount,
-                    criticalImageCount,
-                    created,
-                    images,
-                } = deployment;
-                const isExpanded = expandedRowSet.has(id);
+            <TbodyUnified
```

Comment:
Is the screen recording you shared using the changes in this PR? (https://redhat-internal.slack.com/archives/C02MN2N2UG4/p1717613464474689) I thought there might be an issue with a missing column, but seems like you're setting it to `8`, which is correct. Does it still push the `CVE` column to the far-left when it shows the spinner? That's odd.